.travis.yml

```diff
@@ -1,14 +1,18 @@
+dist: xenial
 language: python
 python:
 - '2.7'
-- '3.4'
 - '3.5'
 - '3.6'
+- '3.7'
 - 'pypy'
```

Comment:
Should we also add `pypy3.5-6.0` since we mention in the README that we support it.

Reply:
Added `pypy3.5` to the list; not sure what `6.0` means here.